splice_machine/src/main/java/com/splicemachine/derby/utils/StatisticsAdmin.java

```diff
@@ -1377,4 +1197,207 @@ private static void dropColumnStatistics(long conglomerateId,
         dd.deleteColumnStatisticsByColumnId(conglomerateId, columnId, tc);
 
     }
+
+    private static class OpenCoreTask implements Callable<StatisticsOperation> {
+
+        private StatisticsOperation statisticsOperation;
+
+        OpenCoreTask(StatisticsOperation statisticsOperation) {
+            this.statisticsOperation = statisticsOperation;
+        }
+
+        @Override
+        public StatisticsOperation call() throws Exception {
+            statisticsOperation.openCore();
+            return statisticsOperation;
+        }
+    }
+
```

Comment:
If OpenCoreTask is not used, please remove it.

Reply:
done